src/debug_toolkit/payloads/stack_trace.py

```diff
@@ -0,0 +1,27 @@
+def entrypoint(output_path: str):
+    import threading
+    import sys
+
+    ALL_THREADS = bool(ALL_THREADS_PLACEHOLDER)
+    stack_trace = get_traceback(ALL_THREADS)
+    for thr in threading.enumerate():
+        if sys.version_info[0] >= 3 and sys.version_info[1] >= 8:
+            #  _native_id is only from python3.8+
+            replacement_str = f"{thr.getName()} tid:{thr._native_id}"
+        else:
+            replacement_str = f"{thr.getName()} thread identity:{thr._ident}"
+        thread_id_str = '0x{:016x}'.format(thr._ident)
+        stack_trace = stack_trace.replace(thread_id_str, replacement_str)
+    stack_trace = stack_trace.replace(thread_id_str, replacement_str)
```

Comment:
isn't 'thread_id_str' defined inside the for loop.
Why are we replacing the last occurrence out side the loop?